scripts/semconv/templates/registry/stable/common.j2

```diff
@@ -0,0 +1,3 @@
+{%- macro print(value) -%}
```

Comment:
If this macro is only used to display enum member values, then in Weaver 0.8, there is a custom Jinja filter that does exactly that: `{{ value | print_member_value }}`.